### PR TITLE
PyPI upload guide

### PR DESCRIPTION
**Pull request by northwestwitch**
Wrote a tutorial on how to upload a Python repo to PyPI

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ For detailed information or if you need to look up specific options there a very
...
 We should be using Pipfile, pipenv.
 
+## Uploading a package to Pypi
+
+In order to be able to install a Python package directly from the pip command, or from the requirements file of another software package, it is necessary to upload the package to [PyPI][PyPI]. A basic tutorial on how to do that is available [here](uploading_to_pypi.md).
+
+
 ## Conda setup
 
 See [conda](conda.md).
```
Comment (henrikstranneheim) [inline python/README.md]: Seems broken

**Comment by northwestwitch** [comment]
The link to conda was indeed broken. I've fixed it now, and I also fixed the link to the non-test PyPI repo page

**Comment by northwestwitch** [comment]
If you guys don't have objections I'd like to merge this document ?!?

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**